packages/runner/src/lib/SecScan.ts

```diff
@@ -61,9 +62,11 @@ export class SecScan {
     );
 
     try {
-      await scan.expect(this._threshold);
+      await scan.expect(this._threshold, { failFast: this._failFast });
 
-      await this.assert(scan);
+      if (this._failFast) {
+        await this.assert(scan);
+      }
```

Comment:
This should not affect assertion at all. `fail-fast` only determines when we consider test is failed, either after first failed assertion or after all.
```suggestion
      await this.assert(scan);
```